pkg/proto/nbcontract/v1/clusterconfig.proto

```diff
@@ -20,18 +35,34 @@ message ClusterConfig {
 }
 
 message ClusterNetworkConfig {
+  // Virtual network name
   string vnet_name = 1;
+
+  // Virtual network resource group
   string vnet_resource_group = 2;
+
+  // Subnet name
   string subnet = 3;
+
+  // Network security group name
   string security_group_name = 4;
+
+  // Route table name
   string route_table = 5;
 }
 
 message LoadBalancerConfig {
-  LoadBalancerSku load_balancer_sku = 1; // default to basic
-  optional bool exclude_master_from_standard_load_balancer = 2; // default to true
-  optional int32 max_load_balancer_rule_count = 3; // default to 148
-  optional bool disable_outbound_snat = 4; // default to false
+  // Load balancer sku, default to basic
+  LoadBalancerSku load_balancer_sku = 1;
+
+  // Specify if master node should be excluded from standard load balancer, default to true 
+  optional bool exclude_master_from_standard_load_balancer = 2;
+
+  // Maximum number of load balancer rules, default to 148
+  optional int32 max_load_balancer_rule_count = 3;
+
+  // Disable outbound SNAT (Source Network Address Translation) for load balancer, default to false
+  optional bool disable_outbound_snat = 4;
```

Comment:
sorry should've asked in an earlier PR, but does this need to be optional? Just checked the getter on the bootstrapper and it seems we want to default to false in the nil case anyway so i dont think we need the nil check.

Reply:
Thanks for catching this. Removed `optional`.